include/ginkgo/core/log/convergence.hpp

```diff
@@ -94,6 +103,13 @@ class Convergence : public Logger {
             new Convergence(exec, enabled_events));
     }
 
+    /**
+     * Returns true if the solver has converged.
```

Comment:
The converge is only true when all solutions are converged, right?

Reply:
yes, but I would say that the solver has converged when only all solutions have converged.